### DISCUSSION TITLE
7주차 스터디 회고

### DISCUSSION THREAD

**Discussion by opgumo3** [Next-Level-Study/effective-java-study#21] [스터디 회고]
한 줄 회고와 건의사항을 말씀해주세요:)

**Comment by opgumo3** [comment]
항등 함수 identity() 메소드가 어떻게 사용될까 싶었는데, 상세한 예제 들고와주셔서 감사합니다! 
Thx to @JeongJaew0n

**Reply by JeongJaew0n** [reply]
감사합니다 
(대충 두레레이 이 허리숙여 인사하는 콘)

**Comment by SuJeong68** [comment]
아직 제네릭 관련 단어를 완벽히 외우지 못해 가끔 헷갈리는 것 같습니다. 설명 들으면서 점점 개념이 잡히고 있는 기분이라 매우 좋습니다. 감사합니다:)

**Comment by JeongJaew0n** [comment]
자바의 정석만 봤을 때는 제네릭을 코드에 적용하기 어려웠고 헷갈리는 부분도 많았는데 이번 장을 보고 나서는 잘 쓸 수 있을 수 있을 것 같습니다!

**Comment by Kurmarkpzgr** [comment]
애너테이션은 많이 어려웠고, Chat GPT는 많이 똑똑했습니다. 앞으로 모르는게 있으면 한 번 써봐야 할거 같습니다.

**Comment by sdh46** [comment]
이번 주는 책에 내용에 대한 이해가 좀 어려워서 평소보다 좀 더 여기저기서 많이 찾아 봤던 것 같습니다. 그리고 개념은 확실히 익혀두는 게 좋을 것 같습니다.

**Comment by narangd0** [comment]
이번 장은 다음 복습 시간을 통해 더 깊이 이해할 수 있으면 좋겠네요
세 번째 발표 수고 많으셨습니다~!
